packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import { compileCss, stripCSSWhitespace } from '../../../../utilities/compileCss';
```

Comment:
Same here with the path? Could you use `./…`

Reply:
Just out of curiosity, are we not able to leverage aliases like with webpack? `@/`, `@icons/`

Reply:
I think we do something similar in `tsconfigs` elsewhere in this repo - let me see if I can do it here. We don't use webpack here @sebastienrocks

Reply:
will update in the next pr!